kratos/python_scripts/time_based_ascii_file_writer_utility.py

```diff
@@ -33,6 +34,7 @@ def __init__(self, model_part, params, file_header):
         # file name and folder path specifications and check
         self.file_name = Path(params["file_name"].GetString())
         self.output_path = Path(params["output_path"].GetString())
+        self.file_extension = params["file_extension"].GetString()
```

Comment:
```suggestion
        self.file_extension = "." + params["file_extension"].GetString()
```

sorry I only realized now that the dot also belongs to the suffix